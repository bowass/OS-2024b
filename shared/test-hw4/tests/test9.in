16060
24169
44869
10951
53659
26357
59531
28516
35483
45584
56975
51036
51836
43555
52482
21788
2367
21918
23903
16386
59913
58268
51662
44248
52383
49855
47856
7452
55008
20601
38658
14297
237
56740
42279
18421
55290
53473
5980
48776
63686
42950
8905
3344
21277
59595
8183
38698
15138
52078
32934
20614
51843
55823
24014
15854
10899
40650
17740
5481
63838
19065
20917
31452
48118
64830
51285
6227
62332
45475
57625
24653
11574
36180
51285
63020
24077
34735
23938
8921
27357
9409
3467
52894
12680
36915
52040
32449
31005
267
14697
9555
40782
36504
27888
12547
49690
23634
15455
11578
3186
4131
15552
33250
1548
983
44801
37724
18429
38881
57414
41852
37773
37659
12531
47204
21181
60626
15992
37809
5829
36492
2459
21362
53843
56917
26602
46750
1727
21066
28142
41195
43510
62886
7639
22180
4292
61380
28773
65140
47205
7856
64579
22793
23671
58970
64645
18830
8947
16485
65022
52967
45643
64967
16166
63510
56853
29158
5063
4022
48269
41185
54256
22905
33786
12576
37304
21873
18049
22020
59302
15602
45987
34771
8724
58476
24043
30081
35191
64561
39754
14243
48783
10197
36676
33781
24675
19148
54367
26729
40608
65204
19847
64931
54554
28613
37992
5826
11378
42535
12244
44455
45549
51062
57585
11219
4189
13702
46413
26177
12013
10527
38113
19294
51739
17547
4445
41011
50788
33531
10697
15267
54485
45220
32061
14025
6133
30641
25983
47368
26568
56082
51361
52091
41956
54536
26123
61121
44356
11565
29332
38195
40132
39569
64675
3860
44552
58885
39084
21762
15209
23641
50652
16845
43753
17309
19493
32072
16126
58837
61954
50763
6882
10399
41944
18266
47141
39259
26982
46702
64147
34071
27199
60381
1683
55083
3158
64507
25179
17006
41261
36373
41734
20305
41670
59432
50971
14489
29604
25461
52352
39207
62523
25397
8106
6589
13839
48183
25047
44894
50060
65241
48742
57570
19375
58212
23961
48771
52460
8160
38076
23882
30301
43163
32419
52972
46941
45135
38354
33048
58379
47002
40680
5578
980
14960
6004
54111
33597
54992
62443
37175
35120
5282
27290
42513
47185
1128
45699
13257
6897
37297
29824
8600
59176
20644
26478
17044
13094
57779
24967
42077
28036
9429
17967
7655
63197
42125
64929
51081
22509
58113
28055
27896
43914
8689
15610
53361
16766
43231
61834
33925
33771
5822
24443
19457
48572
2285
41190
9126
41000
34787
1608
9080
56512
45071
5079
9316
25416
5961
24703
19050
11019
28047
33117
6233
55496
37724
44602
6556
12545
55423
949
33058
1214
3573
58709
1527
41747
33290
17251
44908
52541
40488
37177
37082
37604
36171
36531
17519
21116
36277
34872
18568
59316
57543
30215
20489
45286
20566
29231
62987
29760
43713
2648
48874
55959
62853
47614
29380
13098
63064
63167
35551
322
6544
19002
12366
16775
13178
34483
33733
4949
16452
25555
13802
31764
28740
36878
62874
41861
8847
24321
44554
32378
60330
4163
40795
22455
43957
14248
50234
27926
12562
45965
56510
23203
47851
46547
56514
36785
58243
17517
25827
16905
62041
38735
48026
53389
25837
31854
39534
34693
144
49907
949
54462
25575
51776
8740