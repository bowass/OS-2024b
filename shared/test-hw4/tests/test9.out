Virtual address: 16060 Physical address: 188 Value: 0
Virtual address: 24169 Physical address: 361 Value: 0
Virtual address: 44869 Physical address: 581 Value: 0
Virtual address: 10951 Physical address: 967 Value: -79
Virtual address: 53659 Physical address: 1179 Value: 102
Virtual address: 26357 Physical address: 1525 Value: 0
Virtual address: 59531 Physical address: 1675 Value: 34
Virtual address: 28516 Physical address: 1892 Value: 0
Virtual address: 35483 Physical address: 2203 Value: -90
Virtual address: 45584 Physical address: 2320 Value: 0
Virtual address: 56975 Physical address: 2703 Value: -93
Virtual address: 51036 Physical address: 2908 Value: 0
Virtual address: 51836 Physical address: 3196 Value: 0
Virtual address: 43555 Physical address: 3363 Value: -120
Virtual address: 52482 Physical address: 3586 Value: 51
Virtual address: 21788 Physical address: 3868 Value: 0
Virtual address: 2367 Physical address: 4159 Value: 79
Virtual address: 21918 Physical address: 3998 Value: 21
Virtual address: 23903 Physical address: 4447 Value: 87
Virtual address: 16386 Physical address: 4610 Value: 16
Virtual address: 59913 Physical address: 4873 Value: 0
Virtual address: 58268 Physical address: 5276 Value: 0
Virtual address: 51662 Physical address: 5582 Value: 50
Virtual address: 44248 Physical address: 5848 Value: 0
Virtual address: 52383 Physical address: 6047 Value: 39
Virtual address: 49855 Physical address: 6335 Value: -81
Virtual address: 47856 Physical address: 6640 Value: 0
Virtual address: 7452 Physical address: 6684 Value: 0
Virtual address: 55008 Physical address: 7136 Value: 0
Virtual address: 20601 Physical address: 7289 Value: 0
Virtual address: 38658 Physical address: 7426 Value: 37
Virtual address: 14297 Physical address: 7897 Value: 0
Virtual address: 237 Physical address: 8173 Value: 0
Virtual address: 56740 Physical address: 8356 Value: 0
Virtual address: 42279 Physical address: 8487 Value: 73
Virtual address: 18421 Physical address: 8949 Value: 0
Virtual address: 55290 Physical address: 9210 Value: 53
Virtual address: 53473 Physical address: 9441 Value: 0
Virtual address: 5980 Physical address: 9564 Value: 0
Virtual address: 48776 Physical address: 9864 Value: 0
Virtual address: 63686 Physical address: 10182 Value: 62
Virtual address: 42950 Physical address: 10438 Value: 41
Virtual address: 8905 Physical address: 10697 Value: 0
Virtual address: 3344 Physical address: 10768 Value: 0
Virtual address: 21277 Physical address: 11037 Value: 0
Virtual address: 59595 Physical address: 1739 Value: 50
Virtual address: 8183 Physical address: 11511 Value: -3
Virtual address: 38698 Physical address: 7466 Value: 37
Virtual address: 15138 Physical address: 11554 Value: 14
Virtual address: 52078 Physical address: 11886 Value: 50
Virtual address: 32934 Physical address: 12198 Value: 32
Virtual address: 20614 Physical address: 7302 Value: 20
Virtual address: 51843 Physical address: 3203 Value: -96
Virtual address: 55823 Physical address: 12303 Value: -125
Virtual address: 24014 Physical address: 4558 Value: 23
Virtual address: 15854 Physical address: 12782 Value: 15
Virtual address: 10899 Physical address: 915 Value: -92
Virtual address: 40650 Physical address: 13002 Value: 39
Virtual address: 17740 Physical address: 13132 Value: 0
Virtual address: 5481 Physical address: 13417 Value: 0
Virtual address: 63838 Physical address: 13662 Value: 62
Virtual address: 19065 Physical address: 13945 Value: 0
Virtual address: 20917 Physical address: 14261 Value: 0
Virtual address: 31452 Physical address: 14556 Value: 0
Virtual address: 48118 Physical address: 14838 Value: 46
Virtual address: 64830 Physical address: 14910 Value: 63
Virtual address: 51285 Physical address: 15189 Value: 0
Virtual address: 6227 Physical address: 15443 Value: 20
Virtual address: 62332 Physical address: 15740 Value: 0
Virtual address: 45475 Physical address: 16035 Value: 104
Virtual address: 57625 Physical address: 16153 Value: 0
Virtual address: 24653 Physical address: 16461 Value: 0
Virtual address: 11574 Physical address: 16694 Value: 11
Virtual address: 36180 Physical address: 16980 Value: 0
Virtual address: 51285 Physical address: 15189 Value: 0
Virtual address: 63020 Physical address: 17196 Value: 0
Virtual address: 24077 Physical address: 269 Value: 0
Virtual address: 34735 Physical address: 17583 Value: -21
Virtual address: 23938 Physical address: 4482 Value: 23
Virtual address: 8921 Physical address: 10713 Value: 0
Virtual address: 27357 Physical address: 17885 Value: 0
Virtual address: 9409 Physical address: 18113 Value: 0
Virtual address: 3467 Physical address: 10891 Value: 98
Virtual address: 52894 Physical address: 18334 Value: 51
Virtual address: 12680 Physical address: 18568 Value: 0
Virtual address: 36915 Physical address: 18739 Value: 12
Virtual address: 52040 Physical address: 11848 Value: 0
Virtual address: 32449 Physical address: 19137 Value: 0
Virtual address: 31005 Physical address: 19229 Value: 0
Virtual address: 267 Physical address: 19467 Value: 66
Virtual address: 14697 Physical address: 19817 Value: 0
Virtual address: 9555 Physical address: 20051 Value: 84
Virtual address: 40782 Physical address: 20302 Value: 39
Virtual address: 36504 Physical address: 20632 Value: 0
Virtual address: 27888 Physical address: 20976 Value: 0
Virtual address: 12547 Physical address: 18435 Value: 64
Virtual address: 49690 Physical address: 6170 Value: 48
Virtual address: 23634 Physical address: 21074 Value: 23
Virtual address: 15455 Physical address: 21343 Value: 23
Virtual address: 11578 Physical address: 16698 Value: 11
Virtual address: 3186 Physical address: 21618 Value: 3
Virtual address: 4131 Physical address: 21795 Value: 8
Virtual address: 15552 Physical address: 21440 Value: 0
Virtual address: 33250 Physical address: 22242 Value: 32
Virtual address: 1548 Physical address: 22284 Value: 0
Virtual address: 983 Physical address: 22743 Value: -11
Virtual address: 44801 Physical address: 513 Value: 0
Virtual address: 37724 Physical address: 22876 Value: 0
Virtual address: 18429 Physical address: 8957 Value: 0
Virtual address: 38881 Physical address: 7649 Value: 0
Virtual address: 57414 Physical address: 23110 Value: 56
Virtual address: 41852 Physical address: 23420 Value: 0
Virtual address: 37773 Physical address: 22925 Value: 0
Virtual address: 37659 Physical address: 22811 Value: -58
Virtual address: 12531 Physical address: 23795 Value: 60
Virtual address: 47204 Physical address: 23908 Value: 0
Virtual address: 21181 Physical address: 24253 Value: 0
Virtual address: 60626 Physical address: 24530 Value: 59
Virtual address: 15992 Physical address: 120 Value: 0
Virtual address: 37809 Physical address: 22961 Value: 0
Virtual address: 5829 Physical address: 24773 Value: 0
Virtual address: 36492 Physical address: 20620 Value: 0
Virtual address: 2459 Physical address: 4251 Value: 102
Virtual address: 21362 Physical address: 11122 Value: 20
Virtual address: 53843 Physical address: 24915 Value: -108
Virtual address: 56917 Physical address: 2645 Value: 0
Virtual address: 26602 Physical address: 25322 Value: 25
Virtual address: 46750 Physical address: 25502 Value: 45
Virtual address: 1727 Physical address: 22463 Value: -81
Virtual address: 21066 Physical address: 24138 Value: 20
Virtual address: 28142 Physical address: 25838 Value: 27
Virtual address: 41195 Physical address: 26091 Value: 58
Virtual address: 43510 Physical address: 26358 Value: 42
Virtual address: 62886 Physical address: 26534 Value: 61
Virtual address: 7639 Physical address: 6871 Value: 117
Virtual address: 22180 Physical address: 26788 Value: 0
Virtual address: 4292 Physical address: 21956 Value: 0
Virtual address: 61380 Physical address: 27076 Value: 0
Virtual address: 28773 Physical address: 27237 Value: 0
Virtual address: 65140 Physical address: 27508 Value: 0
Virtual address: 47205 Physical address: 23909 Value: 0
Virtual address: 7856 Physical address: 27824 Value: 0
Virtual address: 64579 Physical address: 27971 Value: 16
Virtual address: 22793 Physical address: 28169 Value: 0
Virtual address: 23671 Physical address: 21111 Value: 29
Virtual address: 58970 Physical address: 28506 Value: 57
Virtual address: 64645 Physical address: 28037 Value: 0
Virtual address: 18830 Physical address: 28814 Value: 18
Virtual address: 8947 Physical address: 10739 Value: -68
Virtual address: 16485 Physical address: 4709 Value: 0
Virtual address: 65022 Physical address: 15102 Value: 63
Virtual address: 52967 Physical address: 18407 Value: -71
Virtual address: 45643 Physical address: 2379 Value: -110
Virtual address: 64967 Physical address: 15047 Value: 113
Virtual address: 16166 Physical address: 28966 Value: 15
Virtual address: 63510 Physical address: 10006 Value: 62
Virtual address: 56853 Physical address: 2581 Value: 0
Virtual address: 29158 Physical address: 29414 Value: 28
Virtual address: 5063 Physical address: 29639 Value: -15
Virtual address: 4022 Physical address: 29878 Value: 3
Virtual address: 48269 Physical address: 30093 Value: 0
Virtual address: 41185 Physical address: 26081 Value: 0
Virtual address: 54256 Physical address: 30448 Value: 0
Virtual address: 22905 Physical address: 28281 Value: 0
Virtual address: 33786 Physical address: 30714 Value: 32
Virtual address: 12576 Physical address: 18464 Value: 0
Virtual address: 37304 Physical address: 30904 Value: 0
Virtual address: 21873 Physical address: 3953 Value: 0
Virtual address: 18049 Physical address: 31105 Value: 0
Virtual address: 22020 Physical address: 26628 Value: 0
Virtual address: 59302 Physical address: 31398 Value: 57
Virtual address: 15602 Physical address: 21490 Value: 15
Virtual address: 45987 Physical address: 31651 Value: -24
Virtual address: 34771 Physical address: 17619 Value: -12
Virtual address: 8724 Physical address: 10516 Value: 0
Virtual address: 58476 Physical address: 31852 Value: 0
Virtual address: 24043 Physical address: 4587 Value: 122
Virtual address: 30081 Physical address: 32129 Value: 0
Virtual address: 35191 Physical address: 32375 Value: 93
Virtual address: 64561 Physical address: 27953 Value: 0
Virtual address: 39754 Physical address: 32586 Value: 38
Virtual address: 14243 Physical address: 7843 Value: -24
Virtual address: 48783 Physical address: 9871 Value: -93
Virtual address: 10197 Physical address: 32981 Value: 0
Virtual address: 36676 Physical address: 33092 Value: 0
Virtual address: 33781 Physical address: 30709 Value: 0
Virtual address: 24675 Physical address: 16483 Value: 24
Virtual address: 19148 Physical address: 14028 Value: 0
Virtual address: 54367 Physical address: 33375 Value: 23
Virtual address: 26729 Physical address: 33641 Value: 0
Virtual address: 40608 Physical address: 12960 Value: 0
Virtual address: 65204 Physical address: 27572 Value: 0
Virtual address: 19847 Physical address: 33927 Value: 97
Virtual address: 64931 Physical address: 15011 Value: 104
Virtual address: 54554 Physical address: 34074 Value: 53
Virtual address: 28613 Physical address: 1989 Value: 0
Virtual address: 37992 Physical address: 34408 Value: 0
Virtual address: 5826 Physical address: 24770 Value: 5
Virtual address: 11378 Physical address: 34674 Value: 11
Virtual address: 42535 Physical address: 34855 Value: -119
Virtual address: 12244 Physical address: 35284 Value: 0
Virtual address: 44455 Physical address: 35495 Value: 105
Virtual address: 45549 Physical address: 16109 Value: 0
Virtual address: 51062 Physical address: 2934 Value: 49
Virtual address: 57585 Physical address: 23281 Value: 0
Virtual address: 11219 Physical address: 35795 Value: -12
Virtual address: 4189 Physical address: 21853 Value: 0
Virtual address: 13702 Physical address: 35974 Value: 13
Virtual address: 46413 Physical address: 36173 Value: 0
Virtual address: 26177 Physical address: 1345 Value: 0
Virtual address: 12013 Physical address: 36589 Value: 0
Virtual address: 10527 Physical address: 36639 Value: 71
Virtual address: 38113 Physical address: 34529 Value: 0
Virtual address: 19294 Physical address: 36958 Value: 18
Virtual address: 51739 Physical address: 3099 Value: -122
Virtual address: 17547 Physical address: 37259 Value: 34
Virtual address: 4445 Physical address: 37469 Value: 0
Virtual address: 41011 Physical address: 25907 Value: 12
Virtual address: 50788 Physical address: 37732 Value: 0
Virtual address: 33531 Physical address: 38139 Value: -66
Virtual address: 10697 Physical address: 36809 Value: 0
Virtual address: 15267 Physical address: 11683 Value: -24
Virtual address: 54485 Physical address: 33493 Value: 0
Virtual address: 45220 Physical address: 38308 Value: 0
Virtual address: 32061 Physical address: 38461 Value: 0
Virtual address: 14025 Physical address: 38857 Value: 0
Virtual address: 6133 Physical address: 9717 Value: 0
Virtual address: 30641 Physical address: 39089 Value: 0
Virtual address: 25983 Physical address: 39295 Value: 95
Virtual address: 47368 Physical address: 39432 Value: 0
Virtual address: 26568 Physical address: 25288 Value: 0
Virtual address: 56082 Physical address: 39698 Value: 54
Virtual address: 51361 Physical address: 15265 Value: 0
Virtual address: 52091 Physical address: 11899 Value: -34
Virtual address: 41956 Physical address: 23524 Value: 0
Virtual address: 54536 Physical address: 34056 Value: 0
Virtual address: 26123 Physical address: 1291 Value: -126
Virtual address: 61121 Physical address: 40129 Value: 0
Virtual address: 44356 Physical address: 35396 Value: 0
Virtual address: 11565 Physical address: 16685 Value: 0
Virtual address: 29332 Physical address: 40340 Value: 0
Virtual address: 38195 Physical address: 40499 Value: 76
Virtual address: 40132 Physical address: 40900 Value: 0
Virtual address: 39569 Physical address: 41105 Value: 0
Virtual address: 64675 Physical address: 28067 Value: 40
Virtual address: 3860 Physical address: 29716 Value: 0
Virtual address: 44552 Physical address: 41224 Value: 0
Virtual address: 58885 Physical address: 28421 Value: 0
Virtual address: 39084 Physical address: 41644 Value: 0
Virtual address: 21762 Physical address: 3842 Value: 21
Virtual address: 15209 Physical address: 11625 Value: 0
Virtual address: 23641 Physical address: 21081 Value: 0
Virtual address: 50652 Physical address: 41948 Value: 0
Virtual address: 16845 Physical address: 42189 Value: 0
Virtual address: 43753 Physical address: 3561 Value: 0
Virtual address: 17309 Physical address: 42397 Value: 0
Virtual address: 19493 Physical address: 42533 Value: 0
Virtual address: 32072 Physical address: 38472 Value: 0
Virtual address: 16126 Physical address: 254 Value: 15
Virtual address: 58837 Physical address: 42965 Value: 0
Virtual address: 61954 Physical address: 43010 Value: 60
Virtual address: 50763 Physical address: 37707 Value: -110
Virtual address: 6882 Physical address: 43490 Value: 6
Virtual address: 10399 Physical address: 43679 Value: 39
Virtual address: 41944 Physical address: 23512 Value: 0
Virtual address: 18266 Physical address: 8794 Value: 17
Virtual address: 47141 Physical address: 23845 Value: 0
Virtual address: 39259 Physical address: 43867 Value: 86
Virtual address: 26982 Physical address: 44134 Value: 26
Virtual address: 46702 Physical address: 25454 Value: 45
Virtual address: 64147 Physical address: 44435 Value: -92
Virtual address: 34071 Physical address: 44567 Value: 69
Virtual address: 27199 Physical address: 17727 Value: -113
Virtual address: 60381 Physical address: 45021 Value: 0
Virtual address: 1683 Physical address: 22419 Value: -92
Virtual address: 55083 Physical address: 9003 Value: -54
Virtual address: 3158 Physical address: 21590 Value: 3
Virtual address: 64507 Physical address: 45307 Value: -2
Virtual address: 25179 Physical address: 45403 Value: -106
Virtual address: 17006 Physical address: 45678 Value: 16
Virtual address: 41261 Physical address: 45869 Value: 0
Virtual address: 36373 Physical address: 20501 Value: 0
Virtual address: 41734 Physical address: 23302 Value: 40
Virtual address: 20305 Physical address: 46161 Value: 0
Virtual address: 41670 Physical address: 46534 Value: 40
Virtual address: 59432 Physical address: 1576 Value: 0
Virtual address: 50971 Physical address: 2843 Value: -58
Virtual address: 14489 Physical address: 46745 Value: 0
Virtual address: 29604 Physical address: 47012 Value: 0
Virtual address: 25461 Physical address: 47221 Value: 0
Virtual address: 52352 Physical address: 6016 Value: 0
Virtual address: 39207 Physical address: 43815 Value: 73
Virtual address: 62523 Physical address: 47419 Value: 14
Virtual address: 25397 Physical address: 47157 Value: 0
Virtual address: 8106 Physical address: 11434 Value: 7
Virtual address: 6589 Physical address: 47805 Value: 0
Virtual address: 13839 Physical address: 38671 Value: -125
Virtual address: 48183 Physical address: 30007 Value: 13
Virtual address: 25047 Physical address: 48087 Value: 117
Virtual address: 44894 Physical address: 606 Value: 43
Virtual address: 50060 Physical address: 48268 Value: 0
Virtual address: 65241 Physical address: 27609 Value: 0
Virtual address: 48742 Physical address: 9830 Value: 47
Virtual address: 57570 Physical address: 23266 Value: 56
Virtual address: 19375 Physical address: 37039 Value: -21
Virtual address: 58212 Physical address: 5220 Value: 0
Virtual address: 23961 Physical address: 4505 Value: 0
Virtual address: 48771 Physical address: 9859 Value: -96
Virtual address: 52460 Physical address: 6124 Value: 0
Virtual address: 8160 Physical address: 11488 Value: 0
Virtual address: 38076 Physical address: 34492 Value: 0
Virtual address: 23882 Physical address: 4426 Value: 23
Virtual address: 30301 Physical address: 48477 Value: 0
Virtual address: 43163 Physical address: 48795 Value: 38
Virtual address: 32419 Physical address: 19107 Value: -88
Virtual address: 52972 Physical address: 18412 Value: 0
Virtual address: 46941 Physical address: 48989 Value: 0
Virtual address: 45135 Physical address: 38223 Value: 19
Virtual address: 38354 Physical address: 40658 Value: 37
Virtual address: 33048 Physical address: 22040 Value: 0
Virtual address: 58379 Physical address: 31755 Value: 2
Virtual address: 47002 Physical address: 49050 Value: 45
Virtual address: 40680 Physical address: 13032 Value: 0
Virtual address: 5578 Physical address: 13514 Value: 5
Virtual address: 980 Physical address: 22740 Value: 0
Virtual address: 14960 Physical address: 49264 Value: 0
Virtual address: 6004 Physical address: 9588 Value: 0
Virtual address: 54111 Physical address: 30303 Value: -41
Virtual address: 33597 Physical address: 30525 Value: 0
Virtual address: 54992 Physical address: 7120 Value: 0
Virtual address: 62443 Physical address: 15851 Value: -6
Virtual address: 37175 Physical address: 30775 Value: 77
Virtual address: 35120 Physical address: 32304 Value: 0
Virtual address: 5282 Physical address: 49570 Value: 5
Virtual address: 27290 Physical address: 17818 Value: 26
Virtual address: 42513 Physical address: 34833 Value: 0
Virtual address: 47185 Physical address: 23889 Value: 0
Virtual address: 1128 Physical address: 49768 Value: 0
Virtual address: 45699 Physical address: 2435 Value: -96
Virtual address: 13257 Physical address: 50121 Value: 0
Virtual address: 6897 Physical address: 43505 Value: 0
Virtual address: 37297 Physical address: 30897 Value: 0
Virtual address: 29824 Physical address: 50304 Value: 0
Virtual address: 8600 Physical address: 50584 Value: 0
Virtual address: 59176 Physical address: 31272 Value: 0
Virtual address: 20644 Physical address: 7332 Value: 0
Virtual address: 26478 Physical address: 25198 Value: 25
Virtual address: 17044 Physical address: 45716 Value: 0
Virtual address: 13094 Physical address: 49958 Value: 12
Virtual address: 57779 Physical address: 16307 Value: 108
Virtual address: 24967 Physical address: 48007 Value: 97
Virtual address: 42077 Physical address: 50781 Value: 0
Virtual address: 28036 Physical address: 25732 Value: 0
Virtual address: 9429 Physical address: 18133 Value: 0
Virtual address: 17967 Physical address: 31023 Value: -117
Virtual address: 7655 Physical address: 6887 Value: 121
Virtual address: 63197 Physical address: 17373 Value: 0
Virtual address: 42125 Physical address: 50829 Value: 0
Virtual address: 64929 Physical address: 15009 Value: 0
Virtual address: 51081 Physical address: 2953 Value: 0
Virtual address: 22509 Physical address: 51181 Value: 0
Virtual address: 58113 Physical address: 5121 Value: 0
Virtual address: 28055 Physical address: 25751 Value: 101
Virtual address: 27896 Physical address: 20984 Value: 0
Virtual address: 43914 Physical address: 51338 Value: 42
Virtual address: 8689 Physical address: 50673 Value: 0
Virtual address: 15610 Physical address: 21498 Value: 15
Virtual address: 53361 Physical address: 9329 Value: 0
Virtual address: 16766 Physical address: 42110 Value: 16
Virtual address: 43231 Physical address: 48863 Value: 55
Virtual address: 61834 Physical address: 51594 Value: 60
Virtual address: 33925 Physical address: 51845 Value: 0
Virtual address: 33771 Physical address: 30699 Value: -6
Virtual address: 5822 Physical address: 24766 Value: 5
Virtual address: 24443 Physical address: 52091 Value: -34
Virtual address: 19457 Physical address: 42497 Value: 0
Virtual address: 48572 Physical address: 52412 Value: 0
Virtual address: 2285 Physical address: 52717 Value: 0
Virtual address: 41190 Physical address: 26086 Value: 40
Virtual address: 9126 Physical address: 52902 Value: 8
Virtual address: 41000 Physical address: 25896 Value: 0
Virtual address: 34787 Physical address: 17635 Value: -8
Virtual address: 1608 Physical address: 22344 Value: 0
Virtual address: 9080 Physical address: 52856 Value: 0
Virtual address: 56512 Physical address: 53184 Value: 0
Virtual address: 45071 Physical address: 38159 Value: 3
Virtual address: 5079 Physical address: 29655 Value: -11
Virtual address: 9316 Physical address: 18020 Value: 0
Virtual address: 25416 Physical address: 47176 Value: 0
Virtual address: 5961 Physical address: 9545 Value: 0
Virtual address: 24703 Physical address: 16511 Value: 31
Virtual address: 19050 Physical address: 13930 Value: 18
Virtual address: 11019 Physical address: 35595 Value: -62
Virtual address: 28047 Physical address: 25743 Value: 99
Virtual address: 33117 Physical address: 22109 Value: 0
Virtual address: 6233 Physical address: 15449 Value: 0
Virtual address: 55496 Physical address: 53448 Value: 0
Virtual address: 37724 Physical address: 22876 Value: 0
Virtual address: 44602 Physical address: 41274 Value: 43
Virtual address: 6556 Physical address: 47772 Value: 0
Virtual address: 12545 Physical address: 18433 Value: 0
Virtual address: 55423 Physical address: 53375 Value: 31
Virtual address: 949 Physical address: 22709 Value: 0
Virtual address: 33058 Physical address: 22050 Value: 32
Virtual address: 1214 Physical address: 49854 Value: 1
Virtual address: 3573 Physical address: 10997 Value: 0
Virtual address: 58709 Physical address: 42837 Value: 0
Virtual address: 1527 Physical address: 53751 Value: 125
Virtual address: 41747 Physical address: 23315 Value: -60
Virtual address: 33290 Physical address: 37898 Value: 32
Virtual address: 17251 Physical address: 42339 Value: -40
Virtual address: 44908 Physical address: 620 Value: 0
Virtual address: 52541 Physical address: 3645 Value: 0
Virtual address: 40488 Physical address: 12840 Value: 0
Virtual address: 37177 Physical address: 30777 Value: 0
Virtual address: 37082 Physical address: 18906 Value: 36
Virtual address: 37604 Physical address: 53988 Value: 0
Virtual address: 36171 Physical address: 16971 Value: 82
Virtual address: 36531 Physical address: 20659 Value: -84
Virtual address: 17519 Physical address: 37231 Value: 27
Virtual address: 21116 Physical address: 24188 Value: 0
Virtual address: 36277 Physical address: 17077 Value: 0
Virtual address: 34872 Physical address: 54072 Value: 0
Virtual address: 18568 Physical address: 54408 Value: 0
Virtual address: 59316 Physical address: 31412 Value: 0
Virtual address: 57543 Physical address: 23239 Value: 49
Virtual address: 30215 Physical address: 48391 Value: -127
Virtual address: 20489 Physical address: 7177 Value: 0
Virtual address: 45286 Physical address: 38374 Value: 44
Virtual address: 20566 Physical address: 7254 Value: 20
Virtual address: 29231 Physical address: 40239 Value: -117
Virtual address: 62987 Physical address: 17163 Value: -126
Virtual address: 29760 Physical address: 50240 Value: 0
Virtual address: 43713 Physical address: 3521 Value: 0
Virtual address: 2648 Physical address: 54616 Value: 0
Virtual address: 48874 Physical address: 9962 Value: 47
Virtual address: 55959 Physical address: 12439 Value: -91
Virtual address: 62853 Physical address: 26501 Value: 0
Virtual address: 47614 Physical address: 39678 Value: 46
Virtual address: 29380 Physical address: 40388 Value: 0
Virtual address: 13098 Physical address: 49962 Value: 12
Virtual address: 63064 Physical address: 17240 Value: 0
Virtual address: 63167 Physical address: 17343 Value: -81
Virtual address: 35551 Physical address: 2271 Value: -73
Virtual address: 322 Physical address: 19522 Value: 0
Virtual address: 6544 Physical address: 47760 Value: 0
Virtual address: 19002 Physical address: 13882 Value: 18
Virtual address: 12366 Physical address: 23630 Value: 12
Virtual address: 16775 Physical address: 42119 Value: 97
Virtual address: 13178 Physical address: 50042 Value: 12
Virtual address: 34483 Physical address: 54963 Value: -84
Virtual address: 33733 Physical address: 30661 Value: 0
Virtual address: 4949 Physical address: 29525 Value: 0
Virtual address: 16452 Physical address: 4676 Value: 0
Virtual address: 25555 Physical address: 47315 Value: -12
Virtual address: 13802 Physical address: 36074 Value: 13
Virtual address: 31764 Physical address: 55060 Value: 0
Virtual address: 28740 Physical address: 27204 Value: 0
Virtual address: 36878 Physical address: 18702 Value: 36
Virtual address: 62874 Physical address: 26522 Value: 61
Virtual address: 41861 Physical address: 23429 Value: 0
Virtual address: 8847 Physical address: 10639 Value: -93
Virtual address: 24321 Physical address: 51969 Value: 0
Virtual address: 44554 Physical address: 41226 Value: 43
Virtual address: 32378 Physical address: 19066 Value: 31
Virtual address: 60330 Physical address: 44970 Value: 58
Virtual address: 4163 Physical address: 21827 Value: 16
Virtual address: 40795 Physical address: 20315 Value: -42
Virtual address: 22455 Physical address: 51127 Value: -19
Virtual address: 43957 Physical address: 51381 Value: 0
Virtual address: 14248 Physical address: 7848 Value: 0
Virtual address: 50234 Physical address: 55354 Value: 49
Virtual address: 27926 Physical address: 25622 Value: 27
Virtual address: 12562 Physical address: 18450 Value: 12
Virtual address: 45965 Physical address: 31629 Value: 0
Virtual address: 56510 Physical address: 53182 Value: 55
Virtual address: 23203 Physical address: 55715 Value: -88
Virtual address: 47851 Physical address: 6635 Value: -70
Virtual address: 46547 Physical address: 36307 Value: 116
Virtual address: 56514 Physical address: 53186 Value: 55
Virtual address: 36785 Physical address: 33201 Value: 0
Virtual address: 58243 Physical address: 5251 Value: -32
Virtual address: 17517 Physical address: 37229 Value: 0
Virtual address: 25827 Physical address: 56035 Value: 56
Virtual address: 16905 Physical address: 45577 Value: 0
Virtual address: 62041 Physical address: 43097 Value: 0
Virtual address: 38735 Physical address: 7503 Value: -45
Virtual address: 48026 Physical address: 14746 Value: 46
Virtual address: 53389 Physical address: 9357 Value: 0
Virtual address: 25837 Physical address: 56045 Value: 0
Virtual address: 31854 Physical address: 55150 Value: 31
Virtual address: 39534 Physical address: 41070 Value: 38
Virtual address: 34693 Physical address: 17541 Value: 0
Virtual address: 144 Physical address: 8080 Value: 0
Virtual address: 49907 Physical address: 6387 Value: -68
Virtual address: 949 Physical address: 22709 Value: 0
Virtual address: 54462 Physical address: 33470 Value: 53
Virtual address: 25575 Physical address: 47335 Value: -7
Virtual address: 51776 Physical address: 3136 Value: 0
Virtual address: 8740 Physical address: 10532 Value: 0
Number of Translated Addresses = 500
Page Faults = 219
Page Fault Rate = 0.438
TLB Hits = 34
TLB Hit Rate = 0.068
